README.rst

```diff
@@ -0,0 +1,183 @@
+.. readit documentation master file, created by
+   sphinx-quickstart on Sun Feb 11 17:08:38 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
```

Comment:
I think we should remove these 4 lines (1 to 4), as this file is not a root file for our `docs`